.gitignore

```diff
@@ -8,3 +8,7 @@ env.d/*.env-*
 
 # Generated Kubernetes configuration file
 .kubeconfig*
+
+k8s/overlays/*
+!k8s/overlays/example
+!k8s/overlays/.template
```

Comment:
I didn't know you can do that. You could also have added the 2 files explicitly.

Reply:
We are using this kind of rules in [other](https://github.com/openfun/marsha/blob/master/.gitignore#L32) [projects](https://github.com/openfun/richie/blob/master/.gitignore#L97).
I like when rules are explicit and match with the reality :grin:

Reply:
My bad, I thought you were French!